src/main/java/io/fabric8/jenkins/openshiftsync/BuildManager.java

```diff
@@ -334,17 +334,17 @@ static void deleteEventToJenkinsJobRun(final Build build) throws Exception {
      * Deletes all job runs that do not have an associated build in OpenShift
      */
     static void reconcileRunsAndBuilds() {
-        logger.fine("Reconciling job runs and builds");
+        logger.info("Reconciling job runs and builds");
         List<WorkflowJob> jobs = Jenkins.getActiveInstance().getAllItems(WorkflowJob.class);
         for (WorkflowJob job : jobs) {
             BuildConfigProjectProperty property = job.getProperty(BuildConfigProjectProperty.class);
             if (property != null) {
                 String ns = property.getNamespace();
                 String name = property.getName();
                 if (StringUtils.isNotBlank(ns) && StringUtils.isNotBlank(name)) {
-                    logger.fine("Checking job " + job + " runs for BuildConfig " + ns + "/" + name);
+                    logger.info("Checking job " + job + " runs for BuildConfig " + ns + "/" + name);
                     OpenShiftClient client = getAuthenticatedOpenShiftClient();
-                    BuildList builds = client.builds().inNamespace(ns).withLabel("buildconfig=" + name).list();
+                    BuildList builds = client.builds().inNamespace(ns).withLabel("openshift.io/build-config.name=" + name).list();
```

Comment:
can we use a constant for this?

Reply:
turns our we already had one ;-) .... switching to that in next push